shakenfist/logutil.py

```diff
@@ -132,14 +155,47 @@ def with_prefix(self, prefix=None):
         return self if prefix is None else SFCustomAdapter(self._logger, self._extra, prefix)
 
     def process(self, msg, kwargs):
-        msg = '%s[%s] %s' % (setproctitle.getproctitle(), os.getpid(), msg)
-        kwargs["extra"] = self.extra
-
         if config.LOG_METHOD_TRACE:
             self._extra['method'] = util_callstack.get_caller(-5)
 
+        # Emit events: set hidden=True in fields to hide this log message from end users
+        if self.extra.get('is_event', False):
+            self._emit_event(msg, self._extra)
```

Comment:
@mandoonandy this is the code we were talking about the other day which decides if a log message is also an event or not. It seems like I changed it to opt in already, but then forgot because this code is so old. The comment is also wrong of course.